gui/slick/interfaces/default/home.mako

```diff
@@ -814,6 +818,15 @@ $(document).ready(function(){
             </script>
         </td>
 
+        <% show_size = 0 %>
+        % try:
+            <% show_size = sickbeard.helpers.get_size(curShow.location) %>
+        % except sickbeard.exceptions.ShowDirNotFoundException:
+            <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        % endtry
+
```

Comment:
I think all of those are wrong, because: https://github.com/SiCKRAGETV/SickRage/blob/master/sickbeard/helpers.py#L1458 doesn't throw `sickbeard.exceptions.ShowDirNotFoundException`